### ISSUE TITLE
Mode template is missing

### ISSUE THREAD

**Issue by MAKIO135** [processing/processing-templates#7]
Maybe a link to https://github.com/martinleopold/TemplateMode should be added?

**Comment by martinleopold** [comment]
Yeah, I built that together with @fjenett for GSOC 2012 (DebugMode). How about integrating this into a new processing-mode-template. Maybe some smaller changes are needed, in naming etc. I'd be happy to do this :)
